tests/test_trie_add.py

```diff
@@ -0,0 +1,78 @@
+from pyeasytries import __version__
+from pyeasytries import Trie
+from pyeasytries.TrieNode import TrieNode
+import pytest
+
+def test_version():
+    assert __version__ == '0.1.0'
+
+def test_add():
+    trie = Trie()
+    assert ((trie.add("hello")) == True)
+
+def test_add_existing_word():
+    trie = Trie()
+    trie.add("hello")
+    assert ((trie.add("hello")) == False)
+
+def test_word_is_string():
```

Comment:
```suggestion
def test_add_word_is_string():
```
Let's add consistent prefixes to all tests function. For all tests related to `add()`, we can add the prefix `test_add`